packages/common-all/src/schema.ts

```diff
@@ -58,6 +60,9 @@ export class SchemaCreationUtils {
               pattern: currPattern.pattern,
             };
           }
+          if (currPattern.desc) {
+            curr["desc"] = currPattern.desc;
```

Comment:
what was this for?

Reply:
To create the desc fields in the auto-generated schema. Since we can't do comments easily, I'm using the desc fields to describe each line in the schema